packages/audioplayers/tizen/src/audio_player.cc

```diff
@@ -211,31 +239,26 @@ void AudioPlayer::SetLatencyMode(bool low_latency) {
 }
 
 int AudioPlayer::GetDuration() {
+  // TODO(seungsoo47): Get duration in milliseconds, returns -1 if no duration
+  // is available. The frontend package will handle these.
   int32_t duration;
   int ret = player_get_duration(player_, &duration);
   if (ret != PLAYER_ERROR_NONE) {
-    throw AudioPlayerError("player_get_duration failed",
-                           get_error_message(ret));
+    OnLog("player_get_duration failed, " + std::string(get_error_message(ret)));
+    return -1;
```

Comment:
um... I don't know what value `player_get_duration()` will return (I don't think millisecond will be less than 0), but is it okay to return -1 even though type is `int *`...?